frontend/tailwind.config.js

```diff
@@ -40,8 +37,13 @@ export default {
       // 'gray': '#8492a6',
       // 'gray-light': '#d3dce6',
       // color palette
+      'alt-green': '#0b5240',
+      'alt-black':'#383838',
+      'border-color': '#3e3e3e', 
+      'alt-grey': '#bdbdbd',
+      'alt-dark': '#5f6368',
       "grey-dark": "#222",
-      "white": "#fff",
+      'white': '#fff !important',
```

Comment:
all these colors will be overwritten by my changes in #2483 so we can merge these, but was there a reason we needed `!important` ? in general a big push behind this refactor is to eliminate all of the bang-importants that are currently littering the styles.

Reply:
We have all the a-links in the `index.css` file defined so it's overriding the tailwind font color on the button. I think once we've replaced all the a-links then we can remove that piece of code and remove the important flag from the color. 

index.css: 
`a:link {
  color: $alt-green;
}`,